test/test.js

```diff
@@ -10,6 +10,12 @@ function sortLeftToRight(a, b) {
 	return 0;
 }
 
+function assertTree(tree, nbElements) {
+	assert.strictEqual(tree._getCount(tree.root), nbElements);
+	assert.strictEqual(tree._isSorted(tree.root), true);
```

Comment:
As mentioned, that's not why I deleted them. I deleted them because they test implementation and not the features of the component making difficult to change the implementation. Even to a valid implementation. And the code is still verifiably correct.